CoreLayersView/src/au/gov/asd/tac/constellation/views/layers/docs/layers-view.html

```diff
@@ -7,6 +7,16 @@
     </head>
     <body>
         <h1>Layers View</h1>
+        <p>
+            The Layers View holds a collection of Layers. Each Layer can represent a static set of elements, or a dynamically 
+            calculated set of elements which match a query criteria. The query structure is based on the expressions framework.
+            <br/>
+            Each layer has a visibility checkbox which determines if that layer is toggled. All elements are on the base layer 0.
+            When one or more check boxes are selected, only the elements that are members of those layers are visibly displayed. 
+            All other elements will have their <strong>visibility</strong> attribute value toggled to 0.0 (invisible).
+            <br/>
+            Arrangements on graphs with a visual schema will be ran on the base layer 0. 
+        </p>
```

Comment:
Are you able to move this paragraph below the table?

Reply:
How does this look? Should it be below the image also?
![image](https://user-images.githubusercontent.com/58677759/123561886-38313480-d7ea-11eb-9e97-e5f57c5d1d54.png)

Reply:
I think I'm ok with the paragraph being above the image as it aligns with what has been done in some of the other pages